e2e/libraries.entity.spec.ts

```diff
@@ -0,0 +1,70 @@
+// back button should go to whereever it came from
+import { expect, test } from '@playwright/test';
+
```

Comment:
You can try to add a config line here to make this file run in parallel which might speed things up, locally anyway

```suggestion
test.describe.configure({
  mode: 'parallel',
});
```